WindowsServerDocs/administration/Linux-Package-Repository-for-Microsoft-Software.md

```diff
@@ -25,7 +25,13 @@ Microsoft's Linux Software Repository is comprised of multiple sub-repositories:
 
 ## Configuring the repositories
 
-Repositories can be configured automatically by installing the Linux package that applies to your Linux distribution and version. The package will install the repository configuration along with the GPG public key used by tools such as apt/yum/zypper to validate the signed packages and/or repository metadata.
+Repositories can be configured automatically by installing the Linux package that applies to your Linux distribution and version. The package will install the repository configuration along with the GPG public key used by tools such as apt, yum or zypper to validate the signed packages and/or repository metadata.
+
+Note that not all supported distributions are listed here, see the currect supported package repositories at [https://packages.microsoft.com/](https://packages.microsoft.com/) and the instructions for [manual configuration](#manual-configuration) below.
+
+### Debian
+
+ - Debian 10 (Buster)<p>`curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -`<p>`sudo apt-add-repository https://packages.microsoft.com/debian/10/prod`<p>`sudo apt-get update`
```

Comment:
Is the ending dash intended in the following command?
```
curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
```
Cosmetic/codestyle: let default level bullet point lists start in column 1 of the line:
```suggestion
- Debian 10 (Buster)<p>`curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -`<p>`sudo apt-add-repository https://packages.microsoft.com/debian/10/prod`<p>`sudo apt-get update`
```

Reply:
Yeah, using the ending dash in `sudo apt-key add -` means that the command reads the output from the previous command so to say.